pegasus/sites.v3/code.org/public/educate/curriculum/elementary-school.md.erb

```diff
@@ -64,6 +64,25 @@ The curriculum has options for teachers  who’d like to incorporate computer sc
 </div>
 <div style="clear: both;"></div>
 
+
+# Continue Learning with Micro:bit Extension Lessons for CS Fundamentals
+
+<div class="col-50" style="padding-right: 20px; float: left; margin-top: 10px">
+<img src="/images/microbit-hands-with-booklets.png" alt="" width="450">
+</div>
+
+
+<div class="col-50" style="margin-top: 10px">
+<div>
+Have you completed CS Fundamentals Course C, D, E or F with your students and are looking to continue your learning? We have partnered with micro:bit to offer 4 extension lessons for each course. These lessons extend the learning students have done during CS Fundamentals while allowing students to try their hand at being a maker and learn about physical computing.
+<br>
+<br>
+<a href="/maker/csf-microbit"><button>Learn more</button></a>
+</div>
+</div>
+<div style="clear: both;"></div>
```

Comment:
Nit –– this code is also part of the CSF page. Is it likely that these will diverge or stay the same? If the same, I'd vote for a partial here, but it's not blocking